pkg/collector/corechecks/cluster/orchestrator/manifest_buffer_test.go

```diff
@@ -83,7 +83,7 @@ func TestOrchestratorManifestBuffer(t *testing.T) {
 // getSender returns a mock Sender
 // When calling OrchestratorManifest, it adds the messges to a global var manifestToSend
 func getSender(t *testing.T) *mocksender.MockSender {
-	sender := mocksender.NewMockSender(check.ID(rune(1)))
+	sender := mocksender.NewMockSender(id.ID(rune(1)))
```

Comment:
same feeback as @clamoriniere `id.ID` is not really explicit

Reply:
Fixed by: https://github.com/DataDog/datadog-agent/pull/17929/commits/a86c685ed917bf8eb9c892e7eb17d9e09aa1f54a